Update zha.markdown with tips on improving Zigbee network range

Hedda opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->

Update zha.markdown with tips on improving Zigbee network range and coverage. Suggest adding this to the ZHA troubleshooting section as one of the more commonly asked questions. 

FYI, as ZHA and other Zigbee implementation is getting popular more and more end-users new to Zigbee are posting in the Home Assistant community forums about issues/problems regarding Zigbee range and stability.

These suggestions advice users on relativly simple actions they can take to greatly improve signal range and mesh network coverage plus lessen possible interference.

These advice suggestions are based on recommendations from experienced Zigbee developers (like Omer Kilic a.k.a. "omerk" who designed the open-source [zzh adapter](https://electrolama.com/projects/zig-a-zig-ah/)) plus several longer Zigbee range discussions threads found all around in the Home Assistant / ZHA / Zigpy and Zigbee2MQTT community forums as well as the[ Zigbee2MQTT guide on how-to improve network range and stability](https://www.zigbee2mqtt.io/how_tos/how_to_improve_network_range_and_stability.html) and other support channels. 

As an example, the most important advice is for users to add one or more "Zigbee Router" devices (normally AC mains powered Zigbee devices) to their home because they they will route/repeat the Zigbee communcation and as such extend network range.

Reference:

* https://community.home-assistant.io/t/advice-on-zigbee-range-extending/175882
* https://community.home-assistant.io/t/zha-is-not-stable-please-help/173203/
* https://community.home-assistant.io/t/building-out-my-zha-network/196709
* https://community.home-assistant.io/t/extend-zigbee-range-coverage-to-reach-distant-xiaomi-aqara-door-sensor/110329/
* https://community.home-assistant.io/t/migrating-from-hue-hub-to-conbee-questions-thoughts-about-coverage-and-extending-range/236569/
* https://community.home-assistant.io/t/best-device-to-act-as-a-zha-zigbee-repeater/103904
* https://community.home-assistant.io/t/understanding-the-zigbee-landscape/94847/
* https://community.home-assistant.io/t/i-am-getting-so-sick-of-zigbee-random-disconnections-basically-every-time-i-reboot/226372/
* https://community.home-assistant.io/t/unable-to-get-home-decorator-hampton-bay-zigbee-fan-to-pair/193657
* https://community.home-assistant.io/t/itead-zigbee-3-0-usb-dongle-stick-only-cost-7-and-is-based-on-silicon-labs-efr32mg21/271144

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: NA
- Link to parent pull request in the Brands repository: NA
- This PR fixes or closes issue: NA

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Hedda committed:
Update zha.markdown with tips on improving Zigbee network range

Update zha.markdown with tips on improving Zigbee network range

Hedda committed:
Mention symptoms with bad USB extension cables

Mention symptoms with bad USB extension cables

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,17 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
 
+### Improving Zigbee network range
+
+One of the more commonly asked questions is in regards to improving or extending signal reception/transmission or resolving problems with poor link quality. The fact is that bad signal reception can be due to many different things and experienced Zigbee users in the community will normally advise that there are a few simple fundamental things that you should really attemt to do first in order to  ...
```
Try splitting this into multiple paragraphs. Long paragraphs like this, are hard to read.

Hedda replied:
Done.

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,17 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+
+1. Add more Zigbee routers between the far away devices and the next nearest router, distributing more Zigbee routers around areas with poorer reception. Zigbee network topology uses a "mesh network" design which means that each device that acts as a Zigbee router will extend the range of your Zigbee network. While there are exceptions, as a rule of thumb, almost all permanently powered devices w ...
+2. Utilize a USB extension cable to move Zigbee coordinator adapter and place your Zigbee coordinator adapter in a sensible position, away from WiFi access-points/routers or other sources of 2.4GHz signals in a location that it is relatively free from strong signal interference. The best location will depend on your building's floorplan though normally you want to place it somewhere in the middle ...
+3. Check your WiFi access-points/routers channels in use to see if your router/access point is overlapping the channel your Zigbee network is using. If so change the channel of the access-points/routers and lock it. It will normally be easier than changing the channel of your Zigbee network and then you will have to re-join/re-pair all of your Zigbee devices. A busy WiFi access-points/routers tha ...
```
Do not use bare URLs in Markdown.

Hedda replied:
Done.

Hedda replied:
Done.

Hedda committed:
Split range extension intro into multiple paragraphs for readability

Split range extension recommendations intro into multiple paragraphs for readability plus use markdown for URL.

Hedda commented:
Split range extension recommendations intro into multiple paragraphs for readability plus use markdown for URL.

Hedda committed:
Add URL to community dicussion about Zigbee range extending devices

Add URL to community dicussion about Zigbee range extending devices

Hedda commented:
@Adminiuga maybe you would like to review this too?

Hedda commented:
Bump

agners commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,21 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
 
+### Improving Zigbee network range
+
+One of the more commonly asked questions about using Zigbee with Home Assistant are regarding to improving or extending signal reception/transmission or resolving problems with poor link quality, thus here are gathered a few possible options to resolve most problems related to range extension. 
```
I'd avoid introduction about commonly asked questions etc. Get right into it: 

```suggestion
Low signal quality can lead to transmission errors and related issues. This section has a list of tips to improve signal quality which maximizes range and resolves most problems related to transmission errors.
```

agners commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,21 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+
+One of the more commonly asked questions about using Zigbee with Home Assistant are regarding to improving or extending signal reception/transmission or resolving problems with poor link quality, thus here are gathered a few possible options to resolve most problems related to range extension. 
+
+The fact is that bad signal reception can be due to many different things and experienced Zigbee users in the community will normally advise that there are a few simple fundamental things that you should really attemt to do first in order to better optimize your Zigbee devices equipment which can resolve many basic problems related to signal strength or interferance issues. Note that there are al ...
```
This is a lot of words with not that much content... Is that paragraph needed? Maybe just adding that last sentence to the paragraph above is enough.

Hedda replied:
Wrote this how I would like to have this topic explained if I was a beginner however please do suggest/make changes.

agners replied:
I moved the important content in the paragraph above (see new review below).

```suggestion
```

agners commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,21 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+
+The fact is that bad signal reception can be due to many different things and experienced Zigbee users in the community will normally advise that there are a few simple fundamental things that you should really attemt to do first in order to better optimize your Zigbee devices equipment which can resolve many basic problems related to signal strength or interferance issues. Note that there are al ...
+
+Just following these simple tips below should help improve signal strength and reduce signal interference which can normally many problems with a Zigbee network.
```
That seems to end up to be quite similar to the first sentence, I think we can drop that too.

```suggestion
```

agners commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,21 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+
+Just following these simple tips below should help improve signal strength and reduce signal interference which can normally many problems with a Zigbee network.
+
+1. Add more Zigbee routers between the far away devices and the next nearest router, distributing more Zigbee routers around areas with poorer reception. Zigbee network topology uses a "mesh network" design which means that each device that acts as a Zigbee router will extend the range of your Zigbee network. While there are exceptions, as a rule of thumb, almost all permanently powered devices w ...
```
> you should know that some end device models from brands like Xiaomi/Aqara have issues connecting through routers.

Interesting, but what should the user do with that information? Is that an advice to not buy Xiami/Aquara devices? Maybe add a link or something here?

Hedda replied:
That specific comment is really only there to act as a warning to be aware that many Xiaomi/Aqara are known to have issues connecting through routers, so the magic bullet to range issues that are routers might not work on all Xiaomi/Aqara devices. 

That is, this is well known in the community  thought that new users with Xiaomi/Aqara devices with range issues should be aware that just adding any router will probably not magically solve all range issues that they have with Xiaomi/Aqara devices. 

There is a related link there already in the previous sentence to https://community.home-assistant.io/t/advice-on-zigbee-range-extending/175882/ which has further discussion and suggestions but thought I should not add the same link again. 

I could write a longer explanation of why this is the case with Xiaomi/Aqara devices but I do not think that I can not write a short explanation about what to do to resolve range issues with Xiaomi/Aqara devices, best is probably to ask in the forum.

agners replied:
I see. Some improvements suggestion:

```suggestion
1. Add more Zigbee routers between devices far away and the next closest router. Distribute more Zigbee routers in areas with poor reception. Zigbee network topology uses a "mesh network" design which means that each device that acts as a Zigbee router will extend the range of your Zigbee network as a whole. While there are exceptions, almost all permanently powered devices serve as a Zigbee router. Adding more permanently powered Zigbee devices allows to get better coverage. There are also dedicated Zigbee routers which you can [find by doing a community search for "Zigbee signal repeater" or "Zigbee range extender"](https://community.home-assistant.io/t/advice-on-zigbee-range-extending/175882/)). Devices that can not act as routers are typically battery operated and known as "end devices". Some end device have issues connecting through routers (e.g. Xiaomi/Aqara devices).
```

agners commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,21 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+Just following these simple tips below should help improve signal strength and reduce signal interference which can normally many problems with a Zigbee network.
+
+1. Add more Zigbee routers between the far away devices and the next nearest router, distributing more Zigbee routers around areas with poorer reception. Zigbee network topology uses a "mesh network" design which means that each device that acts as a Zigbee router will extend the range of your Zigbee network. While there are exceptions, as a rule of thumb, almost all permanently powered devices w ...
+2. Utilize a USB extension cable to move Zigbee coordinator adapter and place your Zigbee coordinator adapter in a sensible position, away from WiFi access-points/routers or other sources of 2.4GHz signals in a location that it is relatively free from strong signal interference. The best location will depend on your building's floorplan though normally you want to place it somewhere in the middle ...
```
```suggestion
2. Use a USB extension cable. This allows to position the Zigbee coordinator adapter for better signal quality. Position the Zigbee coordinator away from WiFi access-points/routers or other sources of 2.4GHz signals to avoid signal interference. The best location depends on your building's floorplan. Ideally you want to place the Zigbee coordinator somewhere in the middle. Building materials do influence signal quality too, as example dense/thick concrete, bricks, masonry, etc. dampen all wireless signals. Place the Zigbee coordinator with some distance from walls, ceilings and floors. Also try different orientation of your Zigbee antenna (or your whole Zigbee coordinator adapter if it has an internal antenna). Some Antennas have a stronger signal in certain direction. Simply changing orientation can improve signal quality already. Note: A bad USB extension cable may lead to connection issues between the system and the Zigbee coordinator adapter, symptoms of this are disconnection messages.
```

agners commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,21 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+
+1. Add more Zigbee routers between the far away devices and the next nearest router, distributing more Zigbee routers around areas with poorer reception. Zigbee network topology uses a "mesh network" design which means that each device that acts as a Zigbee router will extend the range of your Zigbee network. While there are exceptions, as a rule of thumb, almost all permanently powered devices w ...
+2. Utilize a USB extension cable to move Zigbee coordinator adapter and place your Zigbee coordinator adapter in a sensible position, away from WiFi access-points/routers or other sources of 2.4GHz signals in a location that it is relatively free from strong signal interference. The best location will depend on your building's floorplan though normally you want to place it somewhere in the middle ...
+3. Check your WiFi access-points/routers channels in use to see if your router/access point is overlapping the channel your Zigbee network is using. If so change the channel of the access-points/routers and lock it. It will normally be easier than changing the channel of your Zigbee network and then you will have to re-join/re-pair all of your Zigbee devices. A busy WiFi access-points/routers tha ...
```
```suggestion
3. Zigbee and WiFi can operate on various channels in the 2.4GHz spectrum. A busy WiFi access-points/routers that is operating in the same frequency range (overlapping channels) as your Zigbee coordinator will drown out the Zigbee traffic, especially if they are located close to each other. To avoid interference between Zigbee and WiFi try to choose channels without overlap. Check the channel your WiFi access-points/routers is using (either by checking on the routers web interface or using a WiFi analyzer app). The Zigbee channel is currently not shown in the Home Assistant front-end but you can find the channel in the logs (watch out for Network parameters log entry with the channel number, e.g. `radioChannel=15`). Changing the channel of the Zigbee network needs recreating it and re-join/re-pair all of your Zigbee devices. Typically its a lot easier to change the channel used by your WiFi. See [this article for WiFi and Zigbee channels coexistance](https://www.metageek.com/training/resources/zigbee-wifi-coexistence.html) to avoid using overlapping frequency ranges.
```

agners commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,21 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+2. Utilize a USB extension cable to move Zigbee coordinator adapter and place your Zigbee coordinator adapter in a sensible position, away from WiFi access-points/routers or other sources of 2.4GHz signals in a location that it is relatively free from strong signal interference. The best location will depend on your building's floorplan though normally you want to place it somewhere in the middle ...
+3. Check your WiFi access-points/routers channels in use to see if your router/access point is overlapping the channel your Zigbee network is using. If so change the channel of the access-points/routers and lock it. It will normally be easier than changing the channel of your Zigbee network and then you will have to re-join/re-pair all of your Zigbee devices. A busy WiFi access-points/routers tha ...
+4. Update the firmware on your Zigbee coordinator adapter and your Zigbee devices. Note that depending on your hardware the latest Zigbee coordinator firmware might not always be the one recommended by the community so it is advised to ask before upgrading.
+5. If you Zigbee coordinator adapter a removable antenna with an SMA-connector then you have the option of buying a high-gain antenna as an upgrade. In addition, you also have the option to use an antenna extension-cable if needed (though for various reasons that is not a good alternative to only using a USB extension cable with your Zigbee coordinator adapter if possible). This should really onl ...
```
```suggestion
5. If your Zigbee coordinator adapter has a removable antenna (e.g. with an SMA-connector) then you have the option of using a high-gain antenna. Note that antennas with higher gain have directionality: You might have better reception on the same floor, but reception across floors might suffer. In addition, you also have the option to use an antenna extension-cable if needed (usually using just a USB extension cable for your Zigbee coordinator adapter is the better alternative). This should really only be needed if you are trying to cover a long distance, like to another building or very dense/thick walls, ceilings and floors.
```

agners reviewed (changes requested):
Interesting and helpful section :+1: 

Two general tips:
- Try to avoid sentence without a lot of content or meaning. Often such sentence can be just dropped, or made a lot shorter.
- Try to avoid make long sentence. For technical documentation, short, straight forward sentence make things much easier to understand.

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Stefan Agner <stefan@agner.ch>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Stefan Agner <stefan@agner.ch>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Stefan Agner <stefan@agner.ch>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Stefan Agner <stefan@agner.ch>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Stefan Agner <stefan@agner.ch>

agners commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,20 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
 
+### Improving Zigbee network range
+
+Low signal quality can lead to transmission errors and related issues. This section has a list of tips to improve signal quality which maximizes range and resolves most problems related to transmission errors.
```
```suggestion
Low signal quality can lead to transmission errors and related issues. This section has a list of fundamental tips how to improve signal quality. Improving signal quality also maximizes range and resolves most problems related to transmission errors. Please try to follow at least some of this recommendation before posting on the community forums or reporting issues to developers and submitting debug logs.
```

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Stefan Agner <stefan@agner.ch>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Stefan Agner <stefan@agner.ch>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Stefan Agner <stefan@agner.ch>

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,19 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
 
+### Improving Zigbee network range
+
+Low signal quality can lead to transmission errors and related issues. This section has a list of fundamental tips how to improve signal quality. Improving signal quality also maximizes range and resolves most problems related to transmission errors. Please try to follow at least some of this recommendation before posting on the community forums or reporting issues to developers and submitting de ...
```
```suggestion
Low signal quality can lead to transmission errors and related issues. This section has a list of fundamental tips on how to improve signal quality. Improving signal quality also maximizes range and resolves most problems related to transmission errors. Please try to follow at least some of these recommendations before posting on the community forums or reporting issues to developers and submitting debug logs.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,19 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+
+
+
+1. Add more Zigbee routers between devices far away and the next closest router. Distribute more Zigbee routers in areas with poor reception. Zigbee network topology uses a "mesh network" design which means that each device that acts as a Zigbee router will extend the range of your Zigbee network as a whole. While there are exceptions, almost all permanently powered devices serve as a Zigbee rout ...
```
```suggestion
1. Add more Zigbee routers between devices far away and the next closest router. Distribute more Zigbee routers in areas with poor reception. Zigbee network topology uses a "mesh network" design which means that each device that acts as a Zigbee router will extend the range of your Zigbee network as a whole. While there are exceptions, almost all permanently powered devices serve as a Zigbee router. Adding more permanently powered Zigbee devices allows getting better coverage. There are also dedicated Zigbee routers which you can [find by doing a community search for "Zigbee signal repeater" or "Zigbee range extender"](https://community.home-assistant.io/t/advice-on-zigbee-range-extending/175882/)). Devices that can not act as routers are typically battery-operated and known as "end devices". Some end devices have issues connecting through routers (e.g. Xiaomi/Aqara devices).
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,19 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+
+
+1. Add more Zigbee routers between devices far away and the next closest router. Distribute more Zigbee routers in areas with poor reception. Zigbee network topology uses a "mesh network" design which means that each device that acts as a Zigbee router will extend the range of your Zigbee network as a whole. While there are exceptions, almost all permanently powered devices serve as a Zigbee rout ...
+2. Use a USB extension cable. This allows to position the Zigbee coordinator adapter for better signal quality. Position the Zigbee coordinator away from WiFi access-points/routers or other sources of 2.4GHz signals to avoid signal interference. The best location depends on your building's floorplan. Ideally you want to place the Zigbee coordinator somewhere in the middle. Building materials do i ...
```
```suggestion
2. Use a USB extension cable. This allows positioning the Zigbee coordinator adapter for better signal quality. Position the Zigbee coordinator away from Wi-Fi access-points/routers or other sources of 2.4GHz signals to avoid signal interference. The best location depends on your building's floorplan. Ideally, you want to place the Zigbee coordinator somewhere in the middle. Building materials do influence signal quality too, for example, dense/thick concrete, bricks, masonry, etc. dampen all wireless signals. Place the Zigbee coordinator with some distance from walls, ceilings and floors. Also, try different orientations of your Zigbee antenna (or your whole Zigbee coordinator adapter if it has an internal antenna). Some Antennas have a stronger signal in a certain direction. Simply changing orientation can improve signal quality already. Note: A bad USB extension cable may lead to connection issues between the system and the Zigbee coordinator adapter, symptoms of this are disconnection messages.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,19 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+
+1. Add more Zigbee routers between devices far away and the next closest router. Distribute more Zigbee routers in areas with poor reception. Zigbee network topology uses a "mesh network" design which means that each device that acts as a Zigbee router will extend the range of your Zigbee network as a whole. While there are exceptions, almost all permanently powered devices serve as a Zigbee rout ...
+2. Use a USB extension cable. This allows to position the Zigbee coordinator adapter for better signal quality. Position the Zigbee coordinator away from WiFi access-points/routers or other sources of 2.4GHz signals to avoid signal interference. The best location depends on your building's floorplan. Ideally you want to place the Zigbee coordinator somewhere in the middle. Building materials do i ...
+3. Zigbee and WiFi can operate on various channels in the 2.4GHz spectrum. A busy WiFi access-points/routers that is operating in the same frequency range (overlapping channels) as your Zigbee coordinator will drown out the Zigbee traffic, especially if they are located close to each other. To avoid interference between Zigbee and WiFi try to choose channels without overlap. Check the channel you ...
```
```suggestion
3. Zigbee and Wi-Fi can operate on various channels in the 2.4GHz spectrum. A busy Wi-Fi access-points/routers that are operating in the same frequency range (overlapping channels) as your Zigbee coordinator will drown out the Zigbee traffic, especially if they are located close to each other. To avoid interference between Zigbee and Wi-Fi try to choose channels without overlap. Check the channel your Wi-Fi access-points/routers are using (either by checking on the router's web interface or using a Wi-Fi analyzer app). The Zigbee channel is currently not shown in the Home Assistant front-end but you can find the channel in the logs (watch out for Network parameters log entry with the channel number, e.g., `radioChannel=15`). Changing the channel of the Zigbee network needs recreating it and re-join/re-pair all of your Zigbee devices. Typically it's a lot easier to change the channel used by your Wi-Fi. See [this article for Wi-Fi and Zigbee channels coexistance](https://www.metageek.com/training/resources/zigbee-wifi-coexistence.html) to avoid using overlapping frequency ranges.
```

frenck commented on source/_integrations/zha.markdown:
```diff
@@ -242,6 +242,19 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
+2. Use a USB extension cable. This allows to position the Zigbee coordinator adapter for better signal quality. Position the Zigbee coordinator away from WiFi access-points/routers or other sources of 2.4GHz signals to avoid signal interference. The best location depends on your building's floorplan. Ideally you want to place the Zigbee coordinator somewhere in the middle. Building materials do i ...
+3. Zigbee and WiFi can operate on various channels in the 2.4GHz spectrum. A busy WiFi access-points/routers that is operating in the same frequency range (overlapping channels) as your Zigbee coordinator will drown out the Zigbee traffic, especially if they are located close to each other. To avoid interference between Zigbee and WiFi try to choose channels without overlap. Check the channel you ...
+4. Update the firmware on your Zigbee coordinator adapter and your Zigbee devices. Note that depending on your hardware the latest Zigbee coordinator firmware might not always be the one recommended by the community so it is advised to ask before upgrading.
+5. If your Zigbee coordinator adapter has a removable antenna (e.g. with an SMA-connector) then you have the option of using a high-gain antenna. Note that antennas with higher gain have directionality: You might have better reception on the same floor, but reception across floors might suffer. In addition, you also have the option to use an antenna extension-cable if needed (usually using just a ...
```
```suggestion
5. If your Zigbee coordinator adapter has a removable antenna (e.g., with an SMA-connector) then you have the option of using a high-gain antenna. Note that antennas with higher gain have directionality: You might have better reception on the same floor, but reception across floors might suffer. In addition, you also have the option to use an antenna extension-cable if needed (usually using just a USB extension cable for your Zigbee coordinator adapter is the better alternative). This should really only be needed if you are trying to cover a long distance, like to another building or very dense/thick walls, ceilings and floors.
```

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Franck Nijhof <git@frenck.dev>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Franck Nijhof <git@frenck.dev>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Franck Nijhof <git@frenck.dev>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Franck Nijhof <git@frenck.dev>

Hedda committed:
Update source/_integrations/zha.markdown

Co-authored-by: Franck Nijhof <git@frenck.dev>

Hedda commented:
OK now?

Hedda commented:
@frenck @agners Any other changes requested to move this forward?

frenck commented:
@Hedda Will, you stop pinging people already? Thanks 👍
